pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -485,13 +584,13 @@ pub mod pallet {
 				Error::<T>::DomainNotSupported
 			);
 
-			let router =
-				DomainRouters::<T>::get(destination.clone()).ok_or(Error::<T>::RouterNotFound)?;
-
-			router.send(T::Sender::get(), message.clone())?;
+			OutboundMessageQueue::<T>::insert(
+				(destination.clone(), T::Sender::get(), message.clone()),
+				(),
+			);
```

Comment:
I would add the checks for missing routers here and fail in theses cases.

Reply:
Done.